docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,703 @@
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-using-iac-tools"]
+= Using Infrastructure as Code tools with Kafka instances in {product-long-kafka}
+ifdef::context[:parent-context: {context}]
+:context: using-iac-tools
+
+[role="_abstract"]
+As a developer of applications and services, you can use Infrastructure as Code (IaC) tools to automate deployment and management of architectures.
+
+{product-long-kafka} provides two REST APIs, one for accessing the Kafka instance and one for managing resources such as Kafka instances and services accounts. This guide describes how to use different IaC tools to provision and manage your {product-kafka} instances. Information on Kafka resources is provided to IaC tools listed in this guide by the Kafka APIs.
+
+For more information on creating and setting up Kafka instances, see {base-url}{getting-started-url-kafka}[Getting started with {product-long-kafka}^].
+
+
+//Additional line break to resolve mod docs generation error
+
+[id="con-ansible_{context}"]
+== Ansible
+
+[role="_abstract"]
+Ansible is an open source configuration management and automation tool. To learn more about Ansible, including how to install it, see https://www.redhat.com/en/topics/automation/learning-ansible-tutorial[Learning Ansible Basics^].
+
+Ansible automates the management of remote systems and controls their desired state. Ansible works by connecting to your nodes and pushing out small programs - called modules - to these nodes. Modules are the core functionality of Ansible and are used to execute tasks. The output of one module can be the input for the next one or they can be independent of each other.
+
+One or more Ansible modules can be combined to make a play. Two or more plays can be combined to create an Ansible Playbook, a list of sequential tasks (written in YAML) that automatically executes against hosts. Groups of hosts form your Ansible inventory. You can use an Ansible Playbook to describe the desired state of your infrastructure and then have Ansible provision it.
+
+Ansible roles are a special kind of playbook that is fully self-contained and portable with the tasks, variables, configuration templates, and other supporting files that are needed to complete a complex orchestration. Multiple roles can exist inside a collection allowing easy sharing of content through Automation Hub and https://galaxy.ansible.com[Ansible Galaxy^].
+
+//Additional line break to resolve mod docs generation error
+
+[id="proc-install-rhoas-ansible-collection_{context}"]
+=== Installing the RHOAS Ansible collection
+
+[role="_abstract"]
+You can use the https://galaxy.ansible.com/rhoas/rhoas[RHOAS Ansible collection] to fully manage your Kafka environment with Ansible. You can create, retrieve info, configure, deploy, and delete the following {product-long-kafka} resources:
+
+* Kafka instances
+* Service accounts
+* Kafka topics
+* Access Control Lists (ACLs)
+
+.Prerequisites
+
+* Python 3.9 or later is installed.
```

Comment:
Ideally, we'd hyphenate 'Python 3.9', similar to what we've done for other prereqs like this in other guides. https://www.python.org/downloads/ might be a good place to point, but confirm this with an SME...